check_shapes/integration/tf.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2022 The GPflow Contributors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=import-outside-toplevel
+
+import inspect
+from typing import Any, Callable
+
+from ..base_types import Shape
+from ..config import add_is_compiled_mode
+from ..decorator import WrapperPostProcessor, add_wrapper_post_processor
+from ..error_contexts import ErrorContext
+from ..shapes import register_get_shape
+
+
+def install_tf_integration() -> None:
```

Comment:
Maybe have this return whether it worked or not?

```suggestion
def install_tf_integration() -> bool:
```